cafebabel/templates/articles/drafts/edit.html

```diff
@@ -1,49 +1,71 @@
 {% extends "base.html" %}
 
+{% block body_class %}create-article-page{% endblock %}
 {% block body %}
 <form method=post action={{ url_for('drafts.edit', draft_id=article.id) }} enctype=multipart/form-data>
   <input type=hidden name=id value="{{ article.id }}">
-  <div class=text>
+  <div class=file>
+      <label for=image>Image</label>
+      {% if article.has_image %}
+          <img src={{ article_image_url(article) }} alt>
```

Comment:
Should we put a a default `alt` value? Is this W3C valid?

Reply:
Yes, we should, I haven't touched it, i guess you places it like that. Somewhere else, you've written something like `alt="image for {{ article.title }}"`. I think it's better.